### PR TITLE
Release 0.16.0

### PR DESCRIPTION
**Pull request by craicoverflow**
This PR does the following:

- Updated the release version in all packages to 0.16.0
- Change the release branch in `create-graphback`
- Add a 0.16.x section to the website

### CODE REVIEW

**Review by machi1990 — CHANGES_REQUESTED** [review]
Verified this locally and I have the following requested suggestions since the website was not showing 0.16.0 version.

```patch
diff --git a/docs/releases.md b/docs/releases.md
index 30fa0a429..ff30a79db 100644
--- a/docs/releases.md
+++ b/docs/releases.md
@@ -8,7 +8,7 @@ title: Release Notes
 This file contains changes and migration steps for the Graphback project. 
 Please follow individual releases for more information.
 
-# 0.16
+# 0.16.0
 
 ### New Features
 
diff --git a/website/docusaurus.config.js b/website/docusaurus.config.js
index 9529493b4..933211ebc 100644
--- a/website/docusaurus.config.js
+++ b/website/docusaurus.config.js
@@ -32,7 +32,8 @@ module.exports = {
           label: 'Docs',
           position: 'left',
           items: [
-            { label: '0.15.x', to: 'docs/introduction' },
+            { label: '0.16.x', to: 'docs/introduction' },
+            { label: '0.15.x', to: 'docs/0.15.x/introduction' },
             { label: '0.14.x', to: 'docs/0.14.x/introduction' },
             { label: '0.12.x', to: 'docs/0.12.x/gettingstarted' },
             { label: '0.11.x', to: 'docs/0.11.x/gettingstarted' },
@@ -46,7 +47,7 @@ module.exports = {
         },
         {
           to: 'versions',
-          label: `v0.15.x`,
+          label: `v0.16.x`,
           position: 'right',
         },
         {
diff --git a/website/versioned_docs/version-0.16.x/releases.md b/website/versioned_docs/version-0.16.x/releases.md
index 30fa0a429..ff30a79db 100644
--- a/website/versioned_docs/version-0.16.x/releases.md
+++ b/website/versioned_docs/version-0.16.x/releases.md
@@ -8,7 +8,7 @@ title: Release Notes
 This file contains changes and migration steps for the Graphback project. 
 Please follow individual releases for more information.
 
-# 0.16
+# 0.16.0
 
 ### New Features
```

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -32,7 +32,8 @@ module.exports = {
           label: 'Docs',
           position: 'left',
           items: [
-            { label: '0.15.x', to: 'docs/introduction' },
+            { label: '0.16.x', to: 'docs/introduction' },
+            { label: '0.15.x', to: 'docs/0.16.x/introduction' },
```
Comment (machi1990) [inline website/docusaurus.config.js:36]:
```suggestion
            { label: '0.15.x', to: 'docs/0.15.x/introduction' },
```

Comment (machi1990) [reply website/docusaurus.config.js:36]: Added a minor suggestion. 

Can we also update the version in the release note? From `0.16` to `0.16.0`?

Comment (craicoverflow) [reply website/docusaurus.config.js:36]: Added that in https://github.com/aerogear/graphback/pull/1971/commits/56266d28e301367f49788d17b5a86a592ae107f5

Comment (craicoverflow) [reply website/docusaurus.config.js:36]: (Decided it would be good to have it grouped accordingly for merge)

**Review by machi1990 — APPROVED** [review]

**Pull request merged**